src/v/iceberg/manifest_entry_values.cc

```diff
@@ -143,11 +143,11 @@ data_file_content_type content_from_int(int c) {
 iobuf format_to_str(data_file_format f) {
     switch (f) {
     case data_file_format::avro:
-        return iobuf::from("avro");
+        return iobuf::from("AVRO");
```

Comment:
nit: a note here might help for future, but ok to skip to avoid another round of CI.